src/visualizers/P5GLVisualizer.ts

```diff
@@ -0,0 +1,56 @@
+import p5 from 'p5'
+import * as brush from 'p5.brush'
```

Comment:
Cool to have GL

Reply:
Glad you think so, it's been a bit of a pain in the derrière from the point of view of testing. ;)